Description: spelling-error-in-binary usr/bin/mediastream comand command
Bug-Debian: http://lintian.debian.org/tags/spelling-error-in-binary.html

Index: linphone-3.5.2/mediastreamer2/tests/mediastream.c
===================================================================
--- linphone-3.5.2.orig/mediastreamer2/tests/mediastream.c	2012-02-22 22:54:58.000000000 +1100
+++ linphone-3.5.2/mediastreamer2/tests/mediastream.c	2012-06-24 22:19:41.000000000 +1000
@@ -172,7 +172,7 @@
 								"[ --zrtp <secrets file> (enable zrtp) ]\n"
 								"[ --verbose (most verbose messages) ]\n"
 								"[ --video-windows-id <video surface:preview surface>]\n"
-								"[ --srtp <local master_key> <remote master_key> (enable srtp, master key is generated if absent from comand line)\n"
+								"[ --srtp <local master_key> <remote master_key> (enable srtp, master key is generated if absent from command line)\n"
 								"[ --netsim-bandwidth <bandwidth limit in bits/s> (simulates a network download bandwidth limit)\n"
 		;
 
